packages/pg-v5/commands/pull.js

```diff
@@ -125,19 +112,69 @@ const maybeTunnel = function * (herokuDb) {
   return herokuDb
 }
 
+function spawnPipe (commandOne, commandTwo) {
+  return new Promise((resolve, reject) => {
+    commandOne.stdout.on('data', (data) => {
+      commandTwo.stdin.write(data)
+    })
+
+    commandOne.stderr.on('data', (data) => {
```

Comment:
this is doing a few bad things, it's taking off the beginning of the string and mapping stderr to stdout. Just set stderr of both processes to go to file 2

Reply:
I'm not sure if the restore command outputs anything to stdout but if it does it might also be useful to show that

Reply:
@jdxcode both `pg_dump` and `pg_restore` output their `stderr` to the terminal.  For example, here's what gets displayed on the terminal:
```
pg_dump:
reading extensions
pg_dump: identifying extension members
pg_dump: reading schemas
pg_dump: reading user-defined tables
pg_dump: reading user-defined functions
pg_dump: reading user-defined types
pg_dump: reading procedural languages
pg_dump: reading user-defined aggregate functions
pg_dump: reading user-defined operators
pg_dump: reading user-defined access methods
pg_dump: reading user-defined operator classes
pg_dump: reading user-defined operator families
pg_dump: reading user-defined text search parsers
pg_dump: reading user-defined text search templates
pg_dump: reading user-defined text search dictionaries
pg_dump: reading user-defined text search configurations
pg_dump: reading user-defined foreign-data wrappers
pg_dump: reading user-defined foreign servers
pg_dump: reading default privileges
pg_dump: reading user-defined collations
pg_dump: reading user-defined conversions
pg_dump: reading type casts
pg_dump: reading transforms
pg_dump: reading table inheritance information
pg_dump: reading event triggers
pg_dump: finding extension tables
pg_dump: finding inheritance relationships
pg_dump: reading column info for interesting tables
pg_dump: finding the columns and types of table "public.person"
pg_dump: finding the columns and types of table "public.private"
pg_dump: flagging inherited columns in subtables
pg_dump: reading indexes
pg_dump: reading extended statistics
pg_dump: reading constraints
pg_dump: reading triggers
pg_dump: reading rewrite rules
pg_dump: reading policies
pg_dump: reading row security enabled for table "public.person"
pg_dump: reading policies for table "public.person"
pg_dump: reading row security enabled for table "public.private"
pg_dump: reading policies for table "public.private"
pg_dump: reading publications
pg_dump: reading publication membership
pg_dump: reading publication membership for table "public.person"
pg_dump: reading publication membership for table "public.private"
pg_dump: reading subscriptions
pg_dump: reading large objects
pg_dump: reading dependency data
pg_dump: saving encoding = UTF8
pg_dump: saving standard_conforming_strings = on
pg_dump: saving search_path =
pg_dump: saving database definition
pg_dump: dumping contents of table "public.person"
pg_dump: dumping contents of table "public.private"
pg_restore: connecting to database for restore
pg_restore: creating SCHEMA "public"
pg_restore: creating COMMENT "SCHEMA public"
pg_restore: creating EXTENSION "plpgsql"
pg_restore: creating COMMENT "EXTENSION plpgsql"
pg_restore: creating TABLE "public.person"
pg_restore: creating TABLE "public.private"
pg_restore: processing data for table "public.person"
pg_restore: processing data for table "public.private"
heroku-cli: Pulling complete.
```

Reply:
Then this isn't doing anything because console.log is stdout always

Reply:
trim is also not good here because you're not getting back each line necessarily, it could be a partial line. If this was working it would also be double spaced